cli.js

```diff
@@ -138,14 +144,14 @@ program
 
 			function endIfDoneWritingData() {
 
-				if (ended && !numWriting) {
+				if (ended && !numWriting && outputFormat !== 'stdout') {
 					endOutput();
 				}
 			}
 
 			var startOutput = _.once(function() {
 
-				console.log('Writing output to ' + outputFile);
+				console.error('Writing output to ' + outputFile);
```

Comment:
Could probably remove all instances of `console.log` instead of changing them to `console.error` - at least for the case when outputting to stdout.